doc/extract_examples.sh

```diff
@@ -71,7 +71,7 @@ for f in man/man3/*.3monocypher man/man3/optional/*.3monocypher; do
 	# and can't be tested like this
 	if [ ! -h "$f" ] && [ "$f" != "man/man3/crypto_sign_init_first_pass_custom_hash.3monocypher" ]; then
 		echo "// $f"
-		sed -n "/^\.Bd/,/^\.Ed/p" < $f | sed "s/\.Bd.*/{/" | sed "s/\.Ed/}/"
+		cat "$f" | sed -n "/^\.Bd/,/^\.Ed/p" | sed "s/\.Bd.*/{/" | sed "s/\.Ed/}/"
```

Comment:
Useless use of cat? If you want just left to right, you could've moved `<$f` to the beginning of the line instead. Not sure if you were aware of this.

Reply:
Useless cat but easier to understand

I don't think I've ever seen that syntax before so I'll leave it as cat